src/controller/node.c

```diff
@@ -654,6 +668,37 @@ static int node_on_match_proxy_removed(sd_bus_message *m, void *userdata, UNUSED
         return 0;
 }
 
+static char *get_selinux_type(const char *context) {
```

Comment:
If possible, it would be great to have a unit test with a few cases similar to
https://github.com/eclipse-bluechi/bluechi/blob/main/src/controller/test/controller/controller_apply_config_test.c

Maybe we can move it to `libbluechi/selinux-utils.h` so its not a static function and write tests for it?